Sources/Models/ALKConfiguration.swift

```diff
@@ -86,9 +86,6 @@ public struct ALKConfiguration {
     /// Additional information you can pass in message metadata in all the messages.
     public var messageMetadata: [AnyHashable: Any]?
     
-    /// if true then dark mode support is enabled to the SDK.
-    public var isDarkModeEnabled = false
```

Comment:
Why is removed?

Reply:
This is used to verify whether dark mode is enabled. Previously, the check occurred after opening the conversation screen or Conversation List Screen, but for customisations like message bubbles, the check was performed after setting dark mode. This caused an issue with setting a dark color. Now, I am using UIColor to perform the dark mode check, ensuring that the check is already included, and customisations remain unaffected.